docs/src/pages/components/icon/react.mdx

```diff
@@ -19,6 +19,9 @@ import {
   CustomIconExample,
   IconColorExample,
```

Comment:
Looks like we can remove some imports from `@aws-amplify/ui-react`, since they are coming from example files now